paper-checkbox.js

```diff
@@ -85,12 +85,18 @@ const template = html`<style>
     pointer-events: none;
     -webkit-transition: background-color 140ms, border-color 140ms;
     transition: background-color 140ms, border-color 140ms;
+
+    -webkit-transition-duration: var(--paper-checkbox-transition-duration, 140ms, 140ms);
+    transition-duration: var(--paper-checkbox-transition-duration, 140ms, 140ms);
```

Comment:
Can you also remove one of these `140ms`?

Reply:
Done